src/main.py

```diff
@@ -107,9 +107,25 @@ def configure_and_run(args) -> bool:
     if args.visualiser:
         config.load_simulator = True
         config.simulator_graphics = True
+    
+    if args.trainer:
+        config.trainer = True
 
     if args.generator:
         config.generator = True
+
+    if args.debug == 'NONE':
+        config.simulator_write_debug_level = DebugLevel.NONE
+    elif args.debug == 'BASIC':
+        config.simulator_write_debug_level = DebugLevel.BASIC
+    elif args.debug == 'LOW':
+        config.simulator_write_debug_level = DebugLevel.LOW
+    elif args.debug == 'MEDIUM':
+        config.simulator_write_debug_level = DebugLevel.MEDIUM
+    elif args.debug == 'HIGH':
+        config.simulator_write_debug_level = DebugLevel.HIGH
+    else:
+        raise NotImplementedError
```

Comment:
Could we do this with a dictionary? Might be neater, e.g.

```
debug_levels = {"NONE": DebugLevel.None, ...}
config.simulator_write_debug_level = debug_levels[args.debug]
```

Reply:
Better even would be to iterate over the debug levels using the `DebugLevel.[..].__name__` using list comprehension